src/index.html

```diff
@@ -1,13 +1,493 @@
 <!DOCTYPE html>
-<html lang="en">
+<html lang="en" class="page">
   <head>
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <title>Title</title>
+    <title>Eco Cosmetics</title>
     <link rel="stylesheet" href="./styles/main.scss">
+    <link rel="icon" href="images/favicon.png">
   </head>
+
   <body>
-    <h1>Hello Mate Academy</h1>
+    <header class="header page__header">
+      <!-- <nav class="nav">
+        <ul class="nav__list">
+          <li class="nav__item">
+            <a href="" class="nav__link">Shop</a>
+          </li>
+          <li class="nav__item">
+            <a href="" class="nav__link">Contact</a>
+          </li>
+          <li class="nav__item">
+            <a href="" class="nav__link">About</a>
+          </li>
+        </ul>
+      </nav> -->
+
+      <div class="container">
+        <div class="header__content">
+          <a href="#" class="header__menu">
+            <img src="images/menu-open.png" alt="Menu">
+          </a>
+
+          <a href="#" class="header__logo">
+            <img src="images/favicon.png" alt="Logo">
+          </a>
+
+          <a href="#shop" class="header__basket">
+            <img
+              src="images/basket.png"
+              alt="basket"
+              class="basket__logo"
+            >
+          </a>
+        </div>
+      </div>
+
+      <div class="header__part">
+        <div class="header__promo">
+          <picture>
+            <source media="(min-width:1280px)" srcset="images/jars-1280.png">
+            <source media="(min-width:768px)" srcset="images/jars-714.png">
+            <img
+              src="images/jars.png"
+              alt="Cosmetic Photo"
+              class="header__img"
+            >
+          </picture>
+        </div>
+
+        <div class="container header__container">
+          <div class="header__info">
+            <h1 class="title title--lg header__title">
+              Brand of eco-cosmetics and magic candles
+            </h1>
+            <p class="text text--md header__text">20% of sales at a wild bird shelter</p>
+            <a class="header__button button">Shop</a>
```

Comment:
Remember about code style, check it throughout the page
```suggestion
            <p class="text text--md header__text">
              20% of sales at a wild bird shelter
            </p>
            <a class="header__button button">
              Shop
            </a>
```